### ISSUE TITLE
execFile runs out of buffer on macOS 10.13.4

### ISSUE THREAD

**Issue by hult** [giggio/node-chromedriver#150]
chromedriver 2.37 on macOS 10.13.4 has started producing a *lot* of error output for me (didn't happen on macOS 10.13.3). Thousands and thousands of lines like

```
2018-04-18 14:49:16.845 chromedriver[5387:157523] pid(5387)/euid(501) is calling TIS/TSM in non-main thread environment, ERROR : This is NOT allowed. Please call TIS/TSM in main thread!!!
```

Since `lib/chromedriver.js` uses `execFile`, which kills the process after `options.maxBuffer` bytes of output, my tests suddenly stop working.

 `bin/chromedriver` uses another mechanism, and therefore works just fine.

**Comment by hult** [comment]
If I change `execFile` to `spawn`, it works, but only if I pipe stdout/stderr to `process.stdout/stderr`, otherwise, it seems to hang/block somehow?

**Comment by giggio** [comment]
I think it would be better to try to find out why chromedriver is producing so much output, which is not expected. I suggest you file a bug on their issue tracker.
In the mean time, if you can find a way to maintain this package working and still show the necessary output, send a PR and I will be happy to integrate it.

**Comment by hult** [comment]
Yes, the output is probably due to a bug in chromedriver (or in macOS), but it's still kind of a bug in node-chromedriver to crash on too much output too?

This only happens when using `chromedriver.js`, and there, the current release swallows all output (right?). My problem right now is, I can't get `spawn` to work if I don't let it spew all of its output to the parent process's stdout. :)

If I get it to work, I'll create a PR!

**Comment by hult** [comment]
☝️ There you go!

**Comment by giggio** [comment]
Published as 2.38.2
